docs/glossary.md

```diff
@@ -184,7 +184,7 @@ Transpilation
     The transformation of JavaScript code that uses advanced language features, unavailable for some browsers, to code rewritten to support them.
 
 ES6
```

Comment:
Both terms should be listed. "Stacking" the term names as shown below is the syntax in a glossary.

```suggestion
ES6
ECMAScript 6
```